src/lib/persistence.ts

```diff
@@ -0,0 +1,18 @@
+import { IPersistence, IPersistenceRead } from '@rocket.chat/apps-engine/definition/accessors';
+import { RocketChatAssociationModel, RocketChatAssociationRecord } from '@rocket.chat/apps-engine/definition/metadata';
+
+export const persistUIData = async (persistence: IPersistence, id: string, data: any): Promise<void> => {
```

Comment:
avoid using `any` as much as you can, I know in some circumstances is acceptable but some not

Reply:
Sure sir, I will avoid using `any` wherever possible but what is the alternative way here?